docker-compose.yml

```diff
@@ -1,31 +1,40 @@
-version: "3"
+version: '3'
 services:
-  #  Create a service named db.
-  db:
-    image: "postgres:12-alpine"
-    container_name: "pipeline_db"
-    #   Setup the username, password, and database name
+  postgres:
+    image: "postgres"
+    container_name: "postgres"
     environment:
-      - POSTGRES_USER=pipeline
-      - POSTGRES_PASSWORD=pipeline
-      - POSTGRES_DB=pipeline
-    #   Map port 6432 (localhost) to port 5432 on the container
+      POSTGRES_PASSWORD: docker
+      POSTGRES_DB: postgres
+      PG_DATA: /var/lib/postgresql/data/pgdata
     ports:
-      - "6432:5432"
-
-  gitlab:
-    image: 'gitlab/gitlab-ce:latest'
+      - "5432:5432"
     restart: always
+
+  mongo:
+    image: "mongo"
+    container_name: "mongo"
     environment:
-      GITLAB_OMNIBUS_CONFIG: |
-       external_url 'http://localhost:9080'
-       nginx['listen_port'] = 9080                   # make nginx to listen on the same port as confgured in external_url
-       # Add any other gitlab.rb configuration here, each on its own line
+      MONGO_INITDB_ROOT_USERNAME: mongoUser
+      MONGO_INITDB_ROOT_PASSWORD: password
+      MONGO_INITDB_DATABASE: mongo
     ports:
-      - '9080:9080'
-      - '9443:443'
-      - '9022:22'
-    volumes:
-      - '/srv/gitlab/config:/etc/gitlab'
-      - '/srv/gitlab/logs:/var/log/gitlab'
-      - '/srv/gitlab/data:/var/opt/gitlab'
+      - "27017:27017"
+    restart: always
+
+#  gitlab:
+#    image: 'gitlab/gitlab-ce:latest'
+#    restart: always
+#    environment:
+#      GITLAB_OMNIBUS_CONFIG: |
+#       external_url 'http://localhost:9080'
+#       nginx['listen_port'] = 9080                   # make nginx to listen on the same port as confgured in external_url
+#       # Add any other gitlab.rb configuration here, each on its own line
+#    ports:
+#      - '9080:9080'
+#      - '9443:443'
+#      - '9022:22'
+#    volumes:
+#      - '/srv/gitlab/config:/etc/gitlab'
+#      - '/srv/gitlab/logs:/var/log/gitlab'
+#      - '/srv/gitlab/data:/var/opt/gitlab'
```

Comment:
Why do you need it here? Maybe we need some explanation in Readme or todo comment?

Reply:
Agree with Ostap

Reply:
@OstapStrashevskii @GrigoriiBerezin 
because it`s non-official docker compose file was been created for the convenience of developers
but you can discuss it with @ilyaonishenko 
https://jirapct.epam.com/jira/browse/EPMLSTRCMW-168

Reply:
done